public/app/features/alerting/unified/components/admin/ExternalAlertmanagers.tsx

```diff
@@ -70,16 +94,26 @@ export const ExternalAlertmanagers = () => {
     }));
   }, [setModalState]);
 
+  const onChangeAlertmanagerChoice = (alertmanagersChoice: string) => {
+    dispatch(
+      addExternalAlertmanagersAction({ alertmanagers: externalAlertManagers!.map((am) => am.url), alertmanagersChoice })
```

Comment:
Can we replace this with `(externalAlertmanagers ?? [])` or perhaps we should just always return `[]` from `useExternalAmSelector()` 🤔

Reply:
yeah nice